### PR TITLE
docs: update RELEASE.md

### PR DESCRIPTION
**Pull request by kenjis**
**Description**
- update RELEASE.md

**Checklist:**
- [x] Securely signed commits
- [] Component(s) with PHPDoc blocks, only if necessary or adds value
- [] Unit testing, with >80% coverage
- [] User guide updated
- [] Conforms to style guide

### CODE REVIEW

**Review by datamweb — APPROVED** [review]
Thanks🥀

**Review by paulbalandan — APPROVED** [review]

File: admin/RELEASE.md [hunk 1]
```diff
@@ -43,6 +43,7 @@ git clone git@github.com:codeigniter4/userguide.git
 * Set the date in **user_guide_src/source/changelogs/{version}.rst** to format `Release Date: January 31, 2021`
 * Create a new changelog for the next version at **user_guide_src/source/changelogs/{next_version}.rst** and add it to **index.rst**
 * Create **user_guide_src/source/installation/upgrade_{ver}.rst**, fill in the "All Changes" section, and add it to **upgrading.rst**
+    * git diff --name-status master
```
Comment (MGatner) [inline admin/RELEASE.md:46]: It looks like we could use a path spec exclusion here:
```suggestion
    * git diff --name-status master -- . ':!system'
```
https://stackoverflow.com/questions/5685007/making-git-log-ignore-changes-for-certain-paths/21079437#21079437

File: admin/RELEASE.md [hunk 2]
```diff
@@ -53,12 +54,16 @@ git clone git@github.com:codeigniter4/userguide.git
...
     * Version: "v4.x.x"
+    * Target: master
     * Title: "CodeIgniter 4.x.x"
     * Description:
 ```
 CodeIgniter 4.x.x release.
 
 See the changelog: https://github.com/codeigniter4/CodeIgniter4/blob/develop/CHANGELOG.md
+
+## New Contributors
```
Comment (MGatner) [inline admin/RELEASE.md:65]: Thanks for adding this! I've been leaving it when I clear out the rest, I think it's a nice touch to have some "shoutouts".

**Review by MGatner — COMMENTED** [review]
I love the templates!

**Review by MGatner — APPROVED** [review]

**Pull request merged**